### ISSUE TITLE
Please implement --quoted_includes_first for fix_include

### ISSUE THREAD

**Issue by hexagonrecursion** [include-what-you-use/include-what-you-use#1660]
I am trying apply`fix_include` to a project that consistently puts quoted includes first. Unfortunately it appears that (unlike `iwyu`) `fix_include` does not have a `--quoted_includes_first` option

**Comment by hexagonrecursion** [comment]
In other words given this:
#### main.cpp
```c++
#include "foo.h"
#include <vector>
#include <memory>
#include <string>
#include <cassert>
#include "bar.h"

int main()
{
    std::vector<int> v;
    std::unique_ptr<int> p;
    std::string s;
    foo();
    bar();
}
```
#### foo.h
```c++
void foo();
```

#### bar.h
```
void bar();
```

I want to be able to do this:
* Angle includes are not reordered relative to other angle includes and quoted includes not reordered relative to other quoted includes (`fix_include --noreorder`). I want this because the maintainers of the project do not consider the enforcement of alphabetic include order to be worth the hassle
* Quoted includes are first

```diff
>>> Fixing #includes in 'main.cpp'
@@ -1,9 +1,9 @@
 #include "foo.h"
+#include "bar.h"
+
 #include <vector>
 #include <memory>
 #include <string>
-#include <cassert>
-#include "bar.h"
 
 int main()
 {
```

**Comment by hexagonrecursion** [comment]
Currently regardless of whether I pass `-Xiwyu --quoted_includes_first` to iwyu  or not both `fix_include --reorder` and `fix_include --noreorder` put angle includes before quoted includes

Version: include-what-you-use 0.21 based on Ubuntu clang version 17.0.6 (9ubuntu1)

**Comment by kimgr** [comment]
Fixed on master as of 30f1871af415d3e36c74b211b1a84a8c4786afbc.
